File: C/1009.c
#include<stdio.h>


int main(){
	
	char x[10];
	
	float total, y,z;
	
	scanf("%s", &x);
	
	scanf("%f %f", &y, &z);
	
	total = (z*15)/100;
	total= total+y;
	
	printf("TOTAL = R$ %.2f\n", total);
	
	return 0;
	
}

File: Java/1009.java
import java.util.Scanner;

class Main{

    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        String w;
        double x, y, total;

        w = entrada.next();
        x = entrada.nextDouble();
        y = entrada.nextDouble();

        y = (y * 15)/100;
        total = (x+y);
        
        System.out.printf("TOTAL = R$ %.2f\n", total);
    }
}